### PR TITLE
accounts,cmd: fix some typos

### PR DESCRIPTION
**Pull request by levisyin**
Not sure if this typo need to be fixed in line 87975 of signer/fourbyte/4byte.json `"53c210eb": "expectedDebtMissmatch()",`

### CODE REVIEW

File: cmd/geth/main.go [hunk 1]
```diff
@@ -417,7 +417,7 @@ func startNode(ctx *cli.Context, stack *node.Node, backend ethapi.Backend, isCon
 					continue
 				}
 				if timestamp := time.Unix(int64(done.Latest.Time), 0); time.Since(timestamp) < 10*time.Minute {
-					log.Info("Synchronisation completed", "latestnum", done.Latest.Number, "latesthash", done.Latest.Hash(),
```
Comment (MariusVanDerWijden) [inline cmd/geth/main.go:420]: I think this should be Synchronization (or Synchronisation)

File: accounts/keystore/passphrase_test.go [hunk 2]
```diff
@@ -54,7 +54,7 @@ func TestKeyEncryptDecrypt(t *testing.T) {
 		// Recrypt with a new password and start over
 		password += "new data appended" // nolint: gosec
 		if keyjson, err = EncryptKey(key, password, veryLightScryptN, veryLightScryptP); err != nil {
-			t.Errorf("test %d: failed to recrypt key %v", i, err)
+			t.Errorf("test %d: failed to encrypt key %v", i, err)
```
Comment (MariusVanDerWijden) [inline accounts/keystore/passphrase_test.go:57]: I would vote for "re-encrypt" here (and also in the comment on line 54)

File: cmd/geth/main.go (hunk 1 above)
Comment (holiman) [reply cmd/geth/main.go:420]: Please revert this change

File: accounts/keystore/passphrase_test.go (hunk 2 above)
Comment (holiman) [reply accounts/keystore/passphrase_test.go:57]: error-messages for failing tests are very much not important. only us devs ever "consume" them. But sure, "recrypt" is strange, encrypt is better and so is re-encrypt

**Review by holiman — CHANGES_REQUESTED** [review]
Revert one thing, otherwise LGTM, thanks

File: cmd/geth/main.go (hunk 1 above)
Comment (levisyin) [reply cmd/geth/main.go]: Got, I have reverted

File: accounts/keystore/passphrase_test.go (hunk 2 above)
Comment (levisyin) [reply accounts/keystore/passphrase_test.go]: Change to re-encrypt now

**Review by MariusVanDerWijden — APPROVED** [review]
LGTM

**Review by holiman — APPROVED** [review]
LGTM

**Comment by holiman** [comment]
Thanks!

**Pull request merged**